### DISCUSSION TITLE
Bermuda Data

### DISCUSSION THREAD

**Discussion by krdslv** [electricitymaps/electricitymaps-contrib#3318] [Q&A]
Hi everyone.
i'm not a programmer at all but keen on helping with Climate change and ensuring that data is available to my community.

i'm running HA on a Pi but my main computer is Windows and i'd like to contribute but i really don't know what i'm doing.
Can anyone help me with setting this up?
Thanks.

**Comment by VIKTORVAV99** [comment]
I'd be happy to take a crack at building a parser for Bermuda if you can find a live (hourly or better) data source for it. 🙂
